### PR TITLE
Implemented volume deletion by name

### PR DESCRIPTION
**Pull request by omgimanerd**
This pertains to https://github.com/phillbaker/digitalocean-node/issues/11
I'm still exploring the codebase and I'm not entirely sure of the code style.
Let me know how you'd want this implemented. As for fetching volumes by name, I think it might be clear to have an analogous method, even if its redundant.

### CODE REVIEW

**Comment by omgimanerd** [comment]
I haven't tested this yet, and I'll add unit tests in the near future.

**Comment by phillbaker** [comment]
Thanks for getting started on this! Definitely good to add a couple of tests, should be able to adapt the existing ones, this looks like a good start!

**Comment by omgimanerd** [comment]
I've been looking through the various utility functions.
Do you think it would be a better idea to implement `droplets.delete()` in such a way that it takes either a string or object as its first parameter? That way a user can call the same function and pass either an id as a string, or a name/region as an object.

**Comment by phillbaker** [comment]
Hm, ya, actually, maybe updating `Volume.prototype.delete = function(id, callback)` to be something like `Volume.prototype.delete = function(id_or_parameters, callback)`? That makes sense to me!

**Comment by omgimanerd** [comment]
What is the necessity of this unit test?
```

    it('escapes the name', function() {
      testUtils.api.delete('/v2/volumes/foo%2Fbar').reply(204, '');

      client.volumes.delete('foo/bar', function(err) {
        expect(err).to.be.null;
      });
    });
```
It seems to me like `Volume.prototype.delete()` and `Volume.prototype.get()` would never be called with a string parameter. Since I've modified both functions to accept either a string or object as the first parameter, how should I deal with this unit test?

File: README.md [hunk 1]
```diff
@@ -358,7 +358,7 @@ client.domains
 * `client.domains.createRecord(domain.name, attributes, [callback])`
 * `client.domains.getRecord(domain.name, domainRecord.id, [callback])`
 * `client.domains.deleteRecord(domain.name, domainRecord.id, [callback])`
-* `client.domains.updateRecord(domain.name, domainRecord.id,, attributes, [callback])`
+* `client.domains.updateRecord(domain.name, domainRecord.id, attributes, [callback])`
```
Comment (omgimanerd) [inline README.md:361]: Miscellaneous typo here fixed

**Pull request closed without merging**